### ISSUE TITLE
How to know PeriodStart and PeriodEnd on first load

### ISSUE THREAD

**Issue by developernaren** [richardtallent/vue-simple-calendar#98]
I have implemented the `:on-period-change="periodChanged"`. This sends a nice `periodStart` and `periodEnd`. The way my calendar works is that, it loads event within the range by making a remote request. 

But I cannot seem to figure out what date range to request events for on the first load because I cannot seem to know what period the calendar loads. Is there a way to know what range the calendar just loaded on the first load?

**Comment by richardtallent** [comment]
The reason I used the `onPeriodChange` function prop instead of an event is because Vue doesn't emit events during the `mount` method (nor is there an "afterMount" or similar method).

The function will be called when the component updates, and while that gave me an initial execution when I tested it, you're the second person to report the function is not called on the initial update. I haven't had a chance to do more testing, unfortunately.
